application-confluence-migrator-pro-ui/src/main/resources/ConfluenceMigratorPro/ConfluenceBatches/New.xml

```diff
@@ -453,15 +445,25 @@ require(['jquery'], function ($, l10n) {
     <property>
       <code>#confluence-exports {
   list-style: none;
+  margin-top: 1em;
 }
 
 .filtered-out {
   display: none;
 }
-</code>
+
+.checkbox-input-span {
+  display: flex;
+  align-items: center;
+  gap: 0.1em;
+}
+
+input[type="checkbox"] {
```

Comment:
I think it would be better to mark the affected checkboxes with a class and apply this styling to that class instead of doing this.